cmd/agent/app/run.go

```diff
@@ -215,7 +215,7 @@ func StartAgent() error {
 	// start the cmd HTTP server
 	if runtime.GOOS != "android" {
 		if err = api.StartServer(); err != nil {
-			return log.Errorf("Error while starting api server, exiting: %v", err)
+			log.Errorf("Major error while starting api server (API will not be accessible): %v", err)
```

Comment:
Is this something we want on all platforms? We could decide to do this only for Windows?
IMO a bad idea, it's better to have the same behavior on all platforms but I'm just mentioning that we could,  because it's mainly for the Windows installers that we're doing this change.

Reply:
Could be a use-case for `log.Criticalf`?